bin/make-consensus.py

```diff
@@ -268,37 +267,31 @@ def main():
         if args.ivarBedFile:
             tempBamFile = join(tempdir, basename(args.bam) + "-trimmed")
             result = e.execute(
-                "ivar trim -i %r -b %r -p %r -e"
-                % (args.bam, args.ivarBedFile, tempBamFile)
+                f"ivar trim -i {args.bam!r} -b {args.ivarBedFile!r} -p {tempBamFile!r} "
+                "-e"
             )
             ivarTempBamFile = tempBamFile + ".bam"
             sortedIvarTempBamFile = tempBamFile + "-trimmed-sorted.bam"
             result = e.execute(
-                "samtools sort %r -o %r" % (ivarTempBamFile, sortedIvarTempBamFile)
+                f"samtools sort {ivarTempBamFile!r} -o {sortedIvarTempBamFile!r}"
             )
             bamFile = sortedIvarTempBamFile
         else:
             bamFile = args.bam
 
         ivarConsensusFile = join(tempdir, "temporary-consensus")
         result = e.execute(
-            "samtools mpileup -A -Q 0 %r | "
-            "ivar consensus -p %r -q 20 -t %r -m %r"
-            % (
-                bamFile,
-                ivarConsensusFile,
-                args.ivarFrequencyThreshold,
-                args.maskLowCoverage,
-            )
+            f"samtools mpileup -A -Q 0 {bamFile!r} | "
+            f"ivar consensus -p {ivarConsensusFile!r} -q 20 "
+            f"-t {args.ivarFrequencyThreshold!r} -m {args.maskLowCoverage!r}"
         )
 
-        result = e.execute("mv %s %s" % (ivarConsensusFile + ".fa", consensusFile))
+        result = e.execute(f"mv {(ivarConsensusFile + '.fa')!r} {consensusFile!r}")
 
     else:
         result = e.execute(
-            "bcftools consensus --sample '%s' --iupac-codes %s --fasta-ref "
-            "'%s' '%s' > '%s'"
-            % (sample, maskArg, args.reference, vcfFile, consensusFile)
+            f"bcftools consensus --sample {sample!r} --iupac-codes {maskArg} "
```

Comment:
Should be `{maskArg!r}` I guess, though I don't remember if that is a filename.

Reply:
The relevant lines are 
```python
        bedFile = join(tempdir, "mask.bed")
...
        maskArg = "--mask " + bedFile
    else:
        maskArg = ""
```
So I guess the non-quoted is fine.
Also that's what it was in the original, that's why I didn't add `!r`.